src/Components/PokemonList.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+import { useGetListOfPokemonQuery } from '../redux/pokemonApi'
+import Pokemon from './Pokemon'
+
+
+
+const PokemonList = () => {
+	const {data,error,isLoading} =useGetListOfPokemonQuery()
+	const [type, setType] = useState("all")
+
+	const changeType=(type)=>{
+		setType(type)
+	}
+
+	return (
+		<div>
+			{/* <div className='flex'>
+					<button onClick={()=>changeType('all')}>All</button>
+					<button onClick={()=>changeType('water')}>Water</button>
+					<button onClick={()=>changeType('fire')}>Fire</button>
+
+			</div> */}
```

Comment:
- [ ] Please remove commented code as it clutters your codebase 👍

Reply:
Fixed.